pyproject.toml

```diff
@@ -60,9 +60,11 @@ name = "Maxwell G"
 email = "maxwell@gtmx.me"
 
 [project.urls]
+"Documentation" = "https://ansible.readthedocs.io/projects/antsibull-docs/"
 "Source code" = "https://github.com/ansible-community/antsibull-docs"
 "Code of Conduct" = "https://docs.ansible.com/ansible/latest/community/code_of_conduct.html"
 "Bug tracker" = "https://github.com/ansible-community/antsibull-docs/issues"
+"Changelog" = "https://github.com/ansible-community/antsibull-docs/tree/main/CHANGELOG.md"
```

Comment:
LGTM, but I'd consider including the Changelog in the documentation site and linking to that instead. You can symlink `docs/changelog.md -> ../CHANGELOG.md`.

Reply:
I tried that, but it looks like shit, since mkdocs only supports a specific dialect of MarkDown and not CommonMark. IIRC it shows `\` everywhere since the MarkDown dialect it supports doesn't allow to use it for escaping.

(Ref: https://github.com/mkdocs/mkdocs/issues/1835)

Reply:
I opened https://github.com/ansible-community/antsibull-changelog/pull/153. There is still the double changelog (one from mkdocs-material and another from antsibull-changelog), but at least, it's properly formatted.

Reply:
s/changelog/table of contents/